docs/providers/dbt/index.rst

```diff
@@ -0,0 +1,13 @@
+dbt Example DAG
```

Comment:
```suggestion
dbt Cloud Example DAG
```
I think the provider should also be classified under `dbt/cloud` as well. dbt Cloud and dbt Core are vastly different products with very different implementations relative to use in Airflow.

Reply:
Understood, corrected it